spec/controllers/api/v1/book_suggestion_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+describe Api::V1::BookSuggestionController, type: :controller do
+  include_context 'Authenticated User'
+  include Devise::Test::ControllerHelpers
```

Comment:
This line should be in you `rails_hepler.rb`because you will use it in many test.
Also, seems you forgot this line at the top of your file: `require 'rails_helper'`

Reply:
I tried adding the line in `rails_helper.rb` but it was causing issues with the book specs so I added a similar line there `include Devise::Test::IntegrationHelpers` which works for the book specs and book controller specs and kept `include Devise::Test::ControllerHelpers` only in this file.

Reply:
Did you try to add it as that in your `/spec/rails_helpers.rb` ?
```ruby
  config.include Devise::Test::ControllerHelpers, type: :controller
```